drivers/spi/spi_nrfx_spis.c

```diff
@@ -198,6 +228,7 @@ static int transceive(const struct device *dev,
 			nrf_spis_enable(dev_config->spis.p_reg);
 		}
 
+		spi_context_buffers_setup(&dev_data->ctx, tx_bufs, rx_bufs, 1);
```

Comment:
A word of comment would be a nice thing here. It's really not obvious that this call is needed so that `dmm_buffer_in_release()` in `event_handler()` can use `dev_data->ctx.rx_buf`.

Reply:
Removed. Add just assigned to `ctx.rx_buf` in `prepare_for_transfer`.